packages/blockchain-api/src/blockscout.ts

```diff
@@ -58,6 +61,8 @@ export class BlockscoutAPI extends RESTDataSource {
   async getRawTokenTransactions(address: string): Promise<Transaction[]> {
     console.info(`Getting token transactions for address: ${address}`)
 
+    // Measure time at beginning of execution
+    const t0 = performance.now()
```

Comment:
Perhaps this timing measurement could be implemented as a decorator.

Reply:
I felt doing it this way was a little more granular given I didn't understand the structure of the code I was instrumenting as much. I think that would be a good refactor at least.